lib/constructs/sqs.ts

```diff
@@ -0,0 +1,34 @@
+import * as sqs from "aws-cdk-lib/aws-sqs";
+import { QueueProps } from "aws-cdk-lib/aws-sqs";
+import { Construct } from "constructs";
+
+export interface SQSProps extends QueueProps {
+  /**
+   * Props to override the default props of the created SQS
+   */
+  overrideSqsProps?: QueueProps;
+  /**
+   * Environment name to help differentiate the queues made
+   */
+  environmentName: string;
+}
+
+/**
+ * A basic SQS queue
+ * - defaults to an FIFO queue
+ */
+export class Queue extends Construct {
```

Comment:
What do we think about naming our wrapper constructs with an `Atat` prefix? E.g. `AtatQueue` just to be clear that it's including some ATAT-standard configs and behavior?

Reply:
I like it that works for me. I originally had `SQS` and thought that wasn't really good, and `Queue` seemed a little better. But I like `AtatQueue` it seems just right, haha.